### ISSUE TITLE
Ghost tooltip making an appearance

### ISSUE THREAD

**Issue by lynwilhelm** [AllenInstitute/biofile-finder#301]
When I close the add new data source modal, the tooltip for the add button appears like a ghost in the night. Super creepy.

<img width="498" alt="Screenshot 2024-10-23 at 2 04 45 PM" src="https://github.com/user-attachments/assets/30f34219-0082-479b-8ea8-f803b83f64c5">

**Comment by BrianWhitneyAI** [comment]
This is back

**Comment by BrianWhitneyAI** [comment]
Yeah same location. But it seems to be happening on startup. Noticed it when I was working on the modal styling. So maybe reintroduced there
